Code/Engine/Foundation/Configuration/Implementation/Plugin.cpp

```diff
@@ -18,6 +18,12 @@
 ezResult UnloadPluginModule(ezPluginModule& ref_pModule, ezStringView sPluginFile);
 ezResult LoadPluginModule(ezStringView sFileToLoad, ezPluginModule& ref_pModule, ezStringView sPluginFile);
 
+ezDynamicArray<ezString>& GetStaticPlugins()
+{
+  static ezDynamicArray<ezString> s_StaticPlugins;
+  return s_StaticPlugins;
+}
```

Comment:
How is this being used? Do we override that function somehow somewhere?

Reply:
So the process for static linking is like this: `EZ_STATICLINK_LIBRARY` calls `ezPluginRegister` which in turn adds the library to this list here (see bottom of this file).
Finally, `InitializeStaticallyLinkedPlugins` will go through the list of *registered* plugins to add them to the loaded plugin list and any subsequent `LoadPlugin` call will then be able to pick up statically linked plugins.